src/main/java/com/dreamteam/database/EmailService.java

```diff
@@ -38,7 +39,7 @@ public static void sendMessage(Order new_order) throws MessagingException
 		String toEmail = "thedreamteamsoftware+orders@gmail.com";
 		String confirmationMessage = new_order.prettyPrint() +
 		 "Your order has been successfully submitted. Thank you for choosing the Dream Team!";
-
+		
```

Comment:
**Great job on the eleventh hour fixes! 
A small request, can you c&p** 
_confirmationMessage += "\nRecommended Products\n" + OD.findRecommendedProducts(new_order).toString());_
 **to this line and** 
_final OrderDatabase OD = OrderDatabase.getOrders();_
 **anywhere above it in the same scope. Thanks!**

Reply:
Actually, there is a small nuance if the recommended products array returns empty, and in a vast majority of cases, it will; "[]". I say don't worry about it if you just see [] in the email.

Reply:
Added in next commit.